add-on/src/options/forms/gateways-form.js

```diff
@@ -31,7 +32,7 @@ export default function gatewaysForm ({
   const onDisabledOnChange = onOptionChange('disabledOn', hostTextToArray)
   const onEnabledOnChange = onOptionChange('enabledOn', hostTextToArray)
   const mixedContentWarning = !secureContextUrl.test(customGatewayUrl)
-  const supportRedirectToCustomGateway = ipfsNodeType !== 'embedded'
+  const supportRedirectToCustomGateway = POSSIBLE_NODE_TYPES.includes(ipfsNodeType)
```

Comment:
if `POSSIBLE_NODE_TYPES` is set to all the possible node types, then `POSSIBLE_NODE_TYPES.includes(ipfsNodeType)` will always be true, meaning `supportRedirectToCustomGateway` will always be true, right? Do we need a separate `CUSTOM_GATEWAY_SUPPORTING_NODE_TYPES` or should we remove this flag and dependent logic?

Reply:
we should have a list of follow up tasks from MV3 then, in a tracking issue